src/client.ts

```diff
@@ -41,6 +42,7 @@ export class KeycloakAdminClient {
   public clientScopes: ClientScopes;
   public clientPolicies: ClientPolicies;
   public identityProviders: IdentityProviders;
+	public clientRegistrationPolicies : ClientRegistrationPolicies;
```

Comment:
The indentation here seems off, make sure it's aligned with the rest here.
```suggestion
  public clientRegistrationPolicies: ClientRegistrationPolicies;
```

Reply:
thanks, yeah, something strange is happening. I'll fix it.